tasks/vox.rake

```diff
@@ -0,0 +1,52 @@
+def current_version
+  @current_version ||= File.read(File.expand_path('../CHANGELOG.md', __dir__)).match(/^## \[([^\]]+)\]/) { |match| match[1] }
+rescue Errno::ENOENT
+  puts 'Cannot find current version from CHANGELOG.md.  Ignored'
+  @current_version = ''
+end
+
+def next_version
+  today_prefix = Time.now.utc.strftime('%Y-%m-%d')
+
+  if current_version.start_with?(today_prefix)
+    current_version.next
```

Comment:
This will cause problem if we reach YYYY-MM-DD-9 as next will increment DD and wrap the last component to 0.  But 10 releases in a day seems way more that we can probably do for this project, so I do not think it is worth fixing this.